authz/audit/audit_logger.go

```diff
@@ -79,7 +79,7 @@ type Event struct {
 // logger configuration. Concrete types representing configuration of specific
 // audit loggers must embed this interface to implement it.
 type LoggerConfig interface {
-	loggerConfig()
+	LoggerConfig()
```

Comment:
Why did you add an export for this? It's meant to be a dummy interface that we have to explicitly say we are implementing

Reply:
See this comment for more detail: https://github.com/grpc/grpc-go/pull/6230#discussion_r1187692102